# ENPICOM BV
## Maximum Matching code (Python)
In “Perfect Matchings and RNA Secondary Structures”, we considered a problem that required us to assume that every possible nucleotide is involved in base pairing to induce an RNA secondary structure. Yet the only way this could occur is if the frequency of adenine in our RNA strand is equal to the frequency of uracil and if the same holds for guanine and cytosine.

We will therefore begin to explore ways of counting secondary structures in which this condition is not required. A more general combinatorial problem will ask instead for the total number of secondary structures of a strand having a maximum possible number of base pairs.
https://rosalind.info/problems/mmch/
### Getting Started
#### Prerequisites
* Python 3.6+

### Try it Out
```
git clone https://github.com/daniocionini/ENPICOM-BV.git
cd ENPICOM-BV-Application
```
## Error Correction in Reads code (Python)
In “Genome Assembly as Shortest Superstring”, we introduce the problem of assembling a genome from a collection of reads. Even though genome sequencing is a multi-billion dollar enterprise, sequencing machines that identify reads still produce errors a substantial percentage of the time. To make matters worse, these errors are unpredictable; it is difficult to determine if the machine has made an error, let alone where in the read the error has occurred. For this reason, error correction in reads is typically a vital first step in genome assembly.

https://rosalind.info/problems/corr/
### Getting Started
#### Prerequisites
* Python 3.6+

#### Try it Out
```
git clone https://github.com/daniocionini/ENPICOM-BV.git
cd ENPICOM-BV-Application
```
## Needleman-Wunsch algorithm code (Python)
The Needleman–Wunsch algorithm is an algorithm used in bioinformatics to align protein or nucleotide sequences

### Getting Started
#### Prerequisites
* Python 3.6+

#### Try it Out
```
git clone https://github.com/daniocionini/ENPICOM-BV.git
cd ENPICOM-BV-Application
```
